pkg/cmd/cli/install/install.go

```diff
@@ -389,6 +407,23 @@ func (o *Options) Run(c *cobra.Command, f client.Factory) error {
 	if err != nil {
 		return err
 	}
+
+	// Get Kubernetes client for priority class validation
```

Comment:
Will the pod creation fail if users set an invalid priority class? If it will, I don't think we need to do the validation

Reply:
It's only to do a warning log. If you prefer will remove.